spec/services/slack_notification_service_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+describe SlackNotificationService do
+  describe 'initialization' do
+    subject { described_class.new({}) }
+
+    it 'raise an error if no action is set in params' do
+      expect { subject }.to raise_error(ArgumentError)
+    end
+  end
+
+  describe '#send_notification' do
+    subject { described_class.new(params).send_notification }
+
+    let!(:user) do
+      User.create(github_name: 'pepe', blacklisted: false)
+    end
+
+    context 'with valid params' do
+      let(:params) do
+        {
+          github_webhook: { action: described_class::ACTION_METHODS.keys.sample },
+          channel: '#code-review',
```

Comment:
I would put it as a let called "default_channel" or something like that (Or even better you can use the one of the service)

Reply:
oh, `DEFAULT_CHANNEL` ? will try to use it, (or test that it's been used as default)